caffe2/proto/torch.proto

```diff
@@ -416,11 +479,39 @@ message TypeProto {
     optional TensorShapeProto shape = 2;
   }
 
+  // Sequence type: List, Tuple
+  message Sequence {
```

Comment:
I think it would be better to have separate messages for List and Tuple and get rid of Sequence because we know Lists have one elem_type and Tuple have multiple. Is there a specific reason we need to support undefined sequence type?

Reply:
Yeah, because we try to make it onnx compatible. In onnx(-ml) it only supports sequence with the same data type. Let's make it backward compatible. :-)